lib/capistrano/db_sync/postgres/cli.rb

```diff
@@ -4,11 +4,11 @@ def initialize(config)
   end
 
   def dump(to_file, db, options = [])
-    "#{with_pw} pg_dump #{credentials} #{format_args} -f #{to_file} #{options.join(' ')} #{db}"
+    "#{with_pw} pg_dump #{credentials} #{format_args} -f #{to_file} #{options.join(' ')} #{db}".strip
```

Comment:
Agree with `strip`.

The reason why I kept this the classes related to Postgres returning strings as command only is because some of these might run either locally or remotely. To run locally, I'm using Ruby's  `system` method. See [link](https://github.com/heartbits/capistrano-db_sync/blob/master/lib/capistrano/db_sync/executor/local.rb#L14).
Maybe I would be able to use `Command` object if I use `run_locally` in Capistrano to run commands locally? I'll take a deeper look at this. Thanks for being skeptical about this!